Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -75,6 +75,20 @@ def fix_dynamic_linkage
     generic_fix_dynamic_linkage
   end
 
+  def expand_rpath(file, bad_name)
+    suffix = bad_name.sub(/^@rpath/, "")
+
+    # short circuit: we expect lib to be usually correct, so we try it first
+    return (lib + suffix) if (lib + suffix).exist?
```

Comment:
Prefer `lib/suffix` over `lib + suffix` for style.

Reply:
I'm inclined though to remove this short circuit. I don't think it'll make a significant performance difference, and is otherwise unnecessary code.